package.json

```diff
@@ -12,7 +12,7 @@
   "devDependencies": {
     "@vuepress/client": "2.0.0-beta.53",
     "vue": "^3.2.45",
-    "vuepress": "2.0.0-beta.53",
+    "vuepress": "^2.0.0-beta.59",
```

Comment:
Could you skip the modification to `package.json`  and `yarn.lock`, and add them in .gitignore file?

Reply:
These two files are not supposed to be changed.

Reply:
OK!